db/migrate/20190501192018_add_type_to_estimates.rb

```diff
@@ -0,0 +1,15 @@
+class AddTypeToEstimates < ActiveRecord::Migration[5.2]
+  def up
+    execute <<-SQL
+      CREATE TYPE estimate_cost AS ENUM ('ground_transportation', 'lodging', 'meals', 'misc', 'registration', 'rental_vehicle', 'air', 'taxi', 'personal_auto', 'transit_other', 'train');
```

Comment:
Is cost the correct name here?  I would think that was a number.  cost_type maybe?

Reply:
`cost_type` sounds good. `type` alone made rails think I was trying to do single table inheritance 😁